src/settings.js

```diff
@@ -659,7 +659,7 @@ var LZ4 = false;
 // The three options below (DISABLE_EXCEPTION_CATCHING,
 // EXCEPTION_CATCHING_ALLOWED, and DISABLE_EXCEPTION_THROWING) only pertain to
 // Emscripten exception handling and do not control the experimental native wasm
-// exception handling option (EXCEPTION_HANDLING).
+// exception handling option (WASM_EXCEPTIONS).
```

Comment:
This one seem wrong.. it should be mentioning the public option for enabling wasm exceptions, right?

Reply:
Good point. Maybe both?
```suggestion
// exception handling option (-fwasm-exceptions, internal setting: WASM_EXCEPTIONS).
```

Reply:
Done